EfYou/Model/Models/Audit.cs

```diff
@@ -27,7 +27,8 @@ public class Audit
         [Filter(AllowPartialStringMatch = true)]
         public string Type { get; set; }
 
-        public int TypeId { get; set; }
+        [StringLength(36)]
```

Comment:
Is there a reason we use 36 here? Like is it the maximum length of a type or is it like the ultimate question of life, the universe, and everything?

Reply:
36 is a GUID length.

If I don't specify it, by default Entity Framework will create an NVARCHAR(MAX) field... which is fine, but you can't index them in SQL Server.... so conclusion was to set a length that will allow GUID, short, int, long to fit :-)

Reply:
Sounds good